source/includes/_api-reference.md

```diff
@@ -50,18 +58,65 @@ request.AddHeader("Authorization", "Basic [insert credentials here]");
 var request = new RestRequest(Method.GET);
 
 request.AddHeader("Authorization", "ApiKey [insert key here]");
+
+
+// Secure Access Token Authentication
+var request = new RestRequest(Method.GET);
+ 
+request.AddHeader("Authorization", "Bearer [insert Access token here]");
 ```
 
-Intelex utilizes two methods for authentication to the API- Basic and ApiKey.
+Intelex utilizes three methods for authentication to the API- Basic, ApiKey and Secure.
 
 Basic authentication is performed via HTTP Basic Auth and your Intelex user credentials (Username:Password). The credentials are then verified by Intelex and access will be granted or denied accordingly.
 
 ApiKey authentication can be performed by generating API access key(s) from the Intelex User Profile menu (note that this page is only accessible to full-access and admins on the site). 
 The unique generated key can then be used to access the Intelex REST API and used to authenticate and prove that the user is who they claim to be.
 
+Secure token based authentication is supported in V6 API. Only dedicated users can access the API.
+
+During the authentication process, the client provides the ‘client id’ and ‘client secret’ to the authentication endpoint, along with the audience and grant type, as demonstrated above. In response, the API furnishes an access token that comes with a specified time limit. This authentication is exclusively applicable to dedicated users. After successful authentication, users gain access to V6 API endpoints till the token expiration.
+
 All API requests must be made over HTTPS and API requests without authentication will fail. Security on data is managed by the platform security configuration. 
 API requests will maintain the same security settings you have configured in the platform for each user.
 
+
+>auth0 API Endpoint - In the URL the region is the unique region based on the regions they are specifying 
+
+```
+https://intelex-[region]-prd.[region].auth0.com/oauth/token
+```
+>Example Requests:
+
+```CSharp 
+Body for Authentication Endpoint
+{"client_id":"Client ID Data",
+"client_secret":"Client Secret Data",
+"audience":"https://*.intelex.com/v6api",
+"grant_type":"client_credentials"}
+```
+
+>Example Response:
+
+```json
+{
+    "access_token": "Generated Bearer Access Token Used For Validation",
+    "expires_in": "Token Expiry Time In Seconds",
+    "token_type": "Bearer"
+}
+```
+
+### Get secured client id and client secret
+
+Login to the V6 application. Navigate to System Administration, User Administration, and then Navigate to API Access.
+Select the secured API from the API Access page; the client id and client secret will be displayed on the details page. Use these to obtain an access token via the auth0 end point.
+
+### Generate secured access token
+
+Use the auth0 API with the client id, client secret, audience and grant type, the token will be generated with a specified time limit of token expiration.
```

Comment:
Also I do not think we should mention auth0 at all in this document.

Reply:
Removed auth0 from the statement.